doc/visual-programming/source/widgets/unsupervised/distances.rst

```diff
@@ -12,23 +12,23 @@ Signals
 
 -  **Data**
 
-Data set
+A data set
 
 **Outputs**:
 
 -  **Distances**
 
-Distance matrix
+A distance matrix
 
 Description
 -----------
 
-Widget **Distances** computes the distances between either the rows or
-columns in the data set.
+The **Distances** widget computes distances between rows or
+columns in a selected data set.
```

Comment:
I'd say just 'in a data set', since it's not really selected, it's just what's on the input.